pkg/configtx/policies.go

```diff
@@ -173,22 +173,11 @@ func (c *ConfigTx) AddConsortiumOrgPolicy(consortiumName, orgName, policyName st
 
 // RemoveConsortiumOrgPolicy removes an existing policy from an consortiums organization.
 // The removed policy must exist however will not error if it does not exist in configuration.
```

Comment:
Let's reword this line similar to how the other functions above have worded not erroring.

Reply:
Let's also add a note about panicking if the consortiums group, consortium group, or consortium org group doesn't exist